suite-native/module-settings/src/screens/SettingsCustomizationScreen.tsx

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 
-import { Text } from '@suite-native/atoms';
 import { Screen, ScreenHeader } from '@suite-native/navigation';
+import { Box } from '@suite-native/atoms';
+
+import { ColorSchemePicker } from '../components/ColorSchemePicker';
```

Comment:
moved to customization screen, because having theming in localization settings is clearly mistake